roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed

[(roachtest).copy/bank/rows=10000000,nodes=9,txn=true failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1770229&tab=buildLog) on [release-19.1@1fcf7104d19c5c7634cfb52c4302bc9e70c4b9ea](https://github.com/cockroachdb/cockroach/commits/1fcf7104d19c5c7634cfb52c4302bc9e70c4b9ea):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/copy/bank/rows=10000000_nodes=9_txn=true/run_1
	copy.go:126,cluster.go:2344,errgroup.go:57: failed to get default range size: pq: column "raw_config_sql" does not exist

	cluster.go:2368,copy.go:138,copy.go:151,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.registerCopy.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/copy.go:138
		    main.registerCopy.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/copy.go:151
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/copy/bank/rows=10000000,nodes=9,txn=true](https://teamcity.cockroachdb.com/viewLog.html?buildId=1770229&tab=artifacts#/copy/bank/rows=10000000,nodes=9,txn=true)
Related:
- #45380 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45297 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45058 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #27688 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [A-kv-client](https://api.github.com/repos/cockroachdb/cockroach/labels/A-kv-client) [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Acopy%2Fbank%2Frows%3D10000000%2Cnodes%3D9%2Ctxn%3Dtrue.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).copy/bank/rows=10000000,nodes=9,txn=true failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1783217&tab=buildLog) on [release-19.1@d651c17d58e4ce97477e2fa66d04487e9039c1fd](https://github.com/cockroachdb/cockroach/commits/d651c17d58e4ce97477e2fa66d04487e9039c1fd):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200304-1783217/copy/bank/rows=10000000_nodes=9_txn=true/run_1
	copy.go:126,cluster.go:2344,errgroup.go:57: failed to get default range size: pq: column "raw_config_sql" does not exist

	cluster.go:2368,copy.go:138,copy.go:151,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.registerCopy.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/copy.go:138
		    main.registerCopy.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/copy.go:151
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/copy/bank/rows=10000000,nodes=9,txn=true](https://teamcity.cockroachdb.com/viewLog.html?buildId=1783217&tab=artifacts#/copy/bank/rows=10000000,nodes=9,txn=true)
Related:
- #45380 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45297 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45058 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #27688 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [A-kv-client](https://api.github.com/repos/cockroachdb/cockroach/labels/A-kv-client) [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Acopy%2Fbank%2Frows%3D10000000%2Cnodes%3D9%2Ctxn%3Dtrue.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).copy/bank/rows=10000000,nodes=9,txn=true failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1786865&tab=buildLog) on [release-19.1@f19baa3c4a1ce8c9b87842d0187e626aed5d4fb7](https://github.com/cockroachdb/cockroach/commits/f19baa3c4a1ce8c9b87842d0187e626aed5d4fb7):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200305-1786865/copy/bank/rows=10000000_nodes=9_txn=true/run_1
	copy.go:126,cluster.go:2344,errgroup.go:57: failed to get default range size: pq: column "raw_config_sql" does not exist

	cluster.go:2368,copy.go:138,copy.go:151,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.registerCopy.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/copy.go:138
		    main.registerCopy.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/copy.go:151
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/copy/bank/rows=10000000,nodes=9,txn=true](https://teamcity.cockroachdb.com/viewLog.html?buildId=1786865&tab=artifacts#/copy/bank/rows=10000000,nodes=9,txn=true)
Related:
- #45380 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45297 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45058 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #27688 roachtest: copy/bank/rows=10000000,nodes=9,txn=true failed  [A-kv-client](https://api.github.com/repos/cockroachdb/cockroach/labels/A-kv-client) [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Acopy%2Fbank%2Frows%3D10000000%2Cnodes%3D9%2Ctxn%3Dtrue.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/copy.go
+++ b/pkg/cmd/roachtest/copy.go
@@ -164,5 +164,9 @@ func getDefaultRangeSize(
         AS range_max_bytes
 FROM
     [SHOW ZONE CONFIGURATION FOR RANGE default];`).Scan(&rangeMinBytes, &rangeMaxBytes)
+	// Older cluster versions do not contain this column. Use the old default.
+	if err != nil && strings.Contains(err.Error(), `column "raw_config_sql" does not exist`) {
+		rangeMinBytes, rangeMaxBytes, err = 32<<20 /* 32MB */, 64<<20 /* 64MB */, nil
+	}
 	return rangeMinBytes, rangeMaxBytes, err
 }

